src/styles/mo-components/_social.scss

```diff
@@ -69,3 +69,88 @@
     }
   }
 }
+
+.social-btns {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  a.btn {
+    &.social-btn {
+      color: #FFFFFF !important;
+      transition: all 0s !important;
+      text-transform: lowercase !important;
+      padding: 0px;
+    }
+
+    .align-left-third {
+      float: left;
+      width: 25%;
+      text-align: left;
+      height: inherit;
+      position: relative;
+
+      svg {
+        width: 50%;
+        height: 50%;
+        margin-left: 5px;
+        position: absolute;
+        bottom: 25%
+      }
+    }
+
+    .align-center-third {
+      float: left;
+      width: 50%;
+      text-align: center;
+    }
+
+    .align-right-third {
+      float: left;
+      width: 25%;
+      text-align: right;
+    }
+  }
+}
+
+.social-btn {
+  &.fb {
+    background-color: #3B5998;
+  }
+
+  &.fb:hover {
+    background-color: #1877F2;
+  }
+
+  &.twitter {
+    background-color: #1DA1F2;
+  }
+
+  &.twitter:hover {
+    background-color: #3FB6FF;
+  }
+
+  &.ig {
+    background-color: #EA3576;
+  }
+
+  &.ig:hover {
+    background-color: #FF3C82;
+  }
+}
+
+@media (min-width: 1008px) {
+  .max-width-695-button-row {
+    flex: 0 0 30%;
+    max-width: 30%; 
+    height: 38px;
+  }
+}
+
+@media (max-width: 1007px) {
```

Comment:
This really feels like there should be a better way using SASS `col` classes.  And where do the exact pixel requirements come from?  They don't match the `grid-breakpoints` values set in `_variables.scss`

Reply:
Good flag, I agree that this is a bit clumsy. I had written a lot of this css directly in main.css before realizing that file was compiled, hence a lot of hard-coded values. In this case, I chose this value because it matched the breakpoint which was being used to increase the height of the fields in the form on the page, and I wanted the buttons to change at the same page width. It looks like the appropriate variable is actually in `_breakpoints.scss`, not in `_variables.scss`. I will make a task to go back through my .css code and see if I can use existing variables more.

- [ ] Use variables from `_breakpoints.scss` and `_variables.scss`